### PR TITLE
next

### PR DESCRIPTION
**Pull request by farnabaz**
### Features

- Render and verify multiple v2 widgets (#75)
- enable google language auto detection (#72)
- use runtime-config to update module options (#70) 

### 🐛 Bug Fixes 

- Use PascalCase for component registration (#35)
- Catch script init error (#66)
- describe error within execute() (#40)
- remove badge on destroy (#76)

### 📝 Docs & Examples

- Update README, add usage documentation (#74)
- Add example to use both v3 and v2 on the same page (#75)

### 🏡 Chore

- Use `google.com` instead of `recaptcha.net` (#56)

### 💖 Thanks to

- @sgarner (Simon Garner)
- @lat1992 (Mauhoi)
- @owenandrews (Owen Andrews)
- @redbayoub (Red Bayoub)
- @abakermi (Abdelhak Akermi)
- @bason8800

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +58,96 @@ using top level options
 }
 ```
 
+## Runtime config
+
+```js
+{
```
Comment (Atinux) [inline README.md:64]:
```suggestion
// nuxt.config.js
export default {
```

File: README.md [hunk 2]
```diff
@@ -59,6 +58,96 @@ using top level options
...
+<form @submit.prevent="onSubmit">
+  <input autocomplete="true" placeholder="Email" type="email" v-model="email">
+  <input autocomplete="current-password" placeholder="Password" type="password" v-model="password">
+  <recaptcha />
+  <button type="submit">Sign In</button>
+</form>
+```
+
+2. Call `getResponse` inside form submit handler to get reCAPTCHA token:
+```js
```
Comment (Atinux) [inline README.md:90]:
```suggestion

```js
```

File: README.md [hunk 3]
```diff
@@ -59,6 +58,96 @@ using top level options
...
+    console.log('Login error:', error)
+  }
+},
+```
+See: [v2 example](https://github.com/nuxt-community/recaptcha-module/tree/master/example/v2)
+
+### reCAPTCHA v3
+
+1. Call `init` function inside `mounted` hook of your page
+```js
```
Comment (Atinux) [inline README.md:110]:
```suggestion

```js
```

File: README.md [hunk 4]
```diff
@@ -59,6 +58,96 @@ using top level options
...
+  try {
+    await this.$recaptcha.init()
+  } catch (e) {
+    console.error(e);
+  }
+}
+```
+
+2. Call `execute` function form submit handler to get reCAPTCHA token:
+```js
```
Comment (Atinux) [inline README.md:121]:
```suggestion

```js
```

File: README.md [hunk 5]
```diff
@@ -59,6 +58,96 @@ using top level options
...
+beforeDestroy() {
+  this.$recaptcha.destroy()
+}
+```
+
+See: [v3 example](https://github.com/nuxt-community/recaptcha-module/tree/master/example/v3)
+
+
+### Server Side
```
Comment (Atinux) [inline README.md:146]:
```suggestion
### Server Side

```

**Review by Atinux — APPROVED** [review]

**Comment by Atinux** [comment]
@mvrlin do you agree with all those changes?

**Review by mvrlin — APPROVED** [review]

**Comment by mvrlin** [comment]
> @mvrlin do you agree with all those changes?

Yes, thanks :)

**Pull request merged**